back/prisma/schema.prisma

```diff
@@ -10,8 +10,41 @@ datasource db {
 }
 
 model User {
-  id    Int     @default(autoincrement()) @id
-  username  String @unique
-  password  String
-  email     String
+  id       Int    @id @default(autoincrement())
+  username String @unique
+  password String
+  email    String
+}
+
+model Song {
+  id           Int    @id @default(autoincrement())
+  name         String @unique
+  artistId     Int?
+  artist       Artist? @relation(fields: [artistId], references: [id])
+  albumId      Int?
+  album        Album? @relation(fields: [albumId], references: [id])
+  genreId      Int?
+  genre        Genre?  @relation(fields: [genreId], references: [id])
+  difficulties Json
```

Comment:
Please type this.

Reply:
If you can

Reply:
https://github.com/prisma/prisma/issues/3219#issuecomment-670202980